README.md

```diff
@@ -1,4 +1,58 @@
-# nodejs-collection
-A place for the Node.js Collective to organize the editing process
+# The Node.js Collection Team
+A place for the Node.js Collection to organize the technical editing process for articles submitted to the publication.
 
-Note: this team is overseen by the [Node.js Community Committee](https://github.com/nodejs/community-committee)
+## Purpose of Node.js Foundation Medium publication
+The Node.js Foundation’s new Medium publication serves as a channel for the community to broadcast to a wide audience its views related to Node.js and its ecosystem. This may include:
+
+- Summaries of incubated projects or technical decisions made
+- Working project initiatives
+- Use cases and success stories
+- Industry insights and best practices around Node.js and development trends (see topic areas below)
+- Reports on Node.js and JavaScript events (these events must have a code of conduct to be considered)
+- Other topics will certainly be considered, but it should be something of interest to the Node.js community
+
+## Quality
+We are looking for posts that teach and provide value to our community. Contributions should include the meta-narrative that “Node.js is maturing and gaining widespread use in mainstream enterprises, while continuing  to inspire amazing potential and creativity among the massive community using the platform.” Sub-domains and topics for the Medium page include:
```

Comment:
I have some reservations about this section. First, the section does not seem to describe quality; rather, it appears to mostly focus on potential topic or tone. Second, while I agree with how the meta-narrative quotation summarizes the current stage of the Node.js' lifecycle, I'm not sure that it's in the best interest of the community or platform to require that articles "toe the party line." I think the Collection would be better served by focusing on tone/style here rather than prescribing a narrative.

For the sake of discussion, here's another stab this paragraph that 

> ## Tone and Content
>
> We are looking for posts that inform, teach, experiment, and generally enrich our community. There's no hard and fast rule of what an article should or feel like; Node.js Collection articles should be as diverse as the community itself. Ideally, contributions should encourage readers to explore new ideas, dig deeper into old ones, and to inspire readers to experiment and tinker. Articles can cover any number of topics such as:
>
> …
> 
> If you're passionate about something not listed above, please don't hesitate to submit your idea for consideration.